lib/router/router.js

```diff
@@ -182,7 +182,16 @@ function getImageStepsFromRoute(imageStepsStr) {
   return imageSteps.map(function(stepStr) {
     // format: crop=t:15,l:10,w:-10,h:-15
     const stepParts = stepStr.split($this.options.cmdValDelimiter);
-    const shortName = stepParts[0];
+    var shortName = stepParts[0];
+
+    // translate to support gcp image service.
+    if (shortName === "gis" && stepParts[1]) {
+      if (!( parseInt(stepParts[1].substr(1)) === 0 )) {
+        shortName = "rs"
+        const translatedParamValue = translateGISParams.call($this, stepParts[1])
```

Comment:
no need for the temporary variable.

```suggestion
        stepParts[1] = translateGISParams.call($this, stepParts[1])
```